setup.cfg

```diff
@@ -19,3 +19,10 @@ log_cli_level = WARNING
 [flake8]
 max-line-length = 88
 ignore = W503, E121, E126, E211, E225, E501, E203, E402, F401, F811, E231
+
+[mypy]
+ignore_missing_imports = True
```

Comment:
Why is this needed / is it worth adding a comment for it?

Reply:
It's because there are modules without type hints or library stubs. I can add a comment